srcpkgs/intellij-idea-community-edition/template

```diff
@@ -12,6 +12,7 @@ distfiles="https://download.jetbrains.com/idea/ideaIC-${version}-no-jbr.tar.gz"
 checksum=c2e4ac38b93349d0f9454720c58a5b7636b7e6cd04f9bb24008deed026350c8f
 repository=nonfree
 nopie=yes
+conflicts="intellij-idea-ultimate-edition"
```

Comment:
Enough to set in one of packages and would need revision increment.

Reply:
Will you be opposed to leaving it in both?
My intention here is to make both the xbps system aware of the conflict and also let whoever touches the templates know, and if one of the templates doesnt have the `conflicts`, it isnt clear there is a conflict until an install is attempted.

Reply:
Fine, leave it.